luigi/notifications.py

```diff
@@ -157,6 +157,15 @@ def send_email(subject, message, sender, recipients, image_png=None):
         send_email_smtp(config, sender, subject, message, recipients, image_png)
 
 
+def send_notification(subject, message, topic):
```

Comment:
Name is *way* to generic for this patch to be accepted I think. Can you rename it to maybe `send_sns_notification` or something else you find suitable?